pkg/ovncontroller/daemonset.go

```diff
@@ -169,6 +169,24 @@ func CreateOVSDaemonSet(
 	envVars := map[string]env.Setter{}
 	envVars["CONFIG_HASH"] = env.SetValue(configHash)
 
+	initContainers := []corev1.Container{
+		{
+			Name:    "ovsdb-server-init",
+			Command: []string{"/usr/local/bin/container-scripts/init-ovsdb-server.sh"},
+			Image:   instance.Spec.OvsContainerImage,
+			SecurityContext: &corev1.SecurityContext{
+				Capabilities: &corev1.Capabilities{
+					Add:  []corev1.Capability{"NET_ADMIN", "SYS_ADMIN", "SYS_NICE"},
```

Comment:
I don't know if these capabilities are all required for the command; but I guess the PR doesn't escalate anything from what it was doing without the PR, so it should be ok, even if the list could be cleaned up.

Reply:
Yes i just replicated the ones already used